emulate/emulate_queue.py

```diff
@@ -119,10 +120,42 @@ def retrieve_next_animation() -> Code:
 	random_pk = random.choice(pk_list)
 	return Code.objects.get(pk=random_pk)
 
+def is_time_between(begin_time, end_time, check_time=None):
+	# If check time is not given, default to current time
```

Comment:
nazwa funkcji dosc jasno określa co robi, nie ma sensu tutaj na komentarz.
Możesz dodać -> bool żeby było jasne że zwracany typ to bool

Reply:
👍🏻